services/core-web/src/components/common/CoreTable.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
```

Comment:
most important change!

Complete refactor of this file.

Reply:
Very nice refactor!